rq/job.py

```diff
@@ -486,6 +492,9 @@ def to_dict(self, include_meta=True):
             obj['status'] = self._status
         if self._dependency_id is not None:
             obj['dependency_id'] = self._dependency_id
+        _dependents_ids = self.dependent_ids
```

Comment:
`dependent_ids` are stored as a set in `job.dependent_keys` so these don't need to be serialized for storage. I think we can remove these.